test/data/jwt_keys/jwks.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Would prefer this not to be stored in repo

Reply:
It contains only public key. But it is verry likely to remove everything related to the test with JWK url as @nirinchev suggested.

Reply:
Removed.